spi/control-plane/contract-spi/src/main/java/org/eclipse/edc/connector/contract/spi/types/offer/ContractDefinition.java

```diff
@@ -36,9 +38,16 @@
  */
 @JsonDeserialize(builder = ContractDefinition.Builder.class)
 public class ContractDefinition extends Entity {
+
+    /**
+     * If not specified, each {@link ContractOffer} defines a 1-year contract.
+     */
+    private static final long DEFAULT_CONTRACT_VALIDITY_DURATION = TimeUnit.DAYS.toSeconds(365);
+
     private String accessPolicyId;
     private String contractPolicyId;
     private AssetSelectorExpression selectorExpression;
+    private long contractValidityDuration = DEFAULT_CONTRACT_VALIDITY_DURATION;
```

Comment:
`contract` is implicit, and `validity` and `duration` are pretty much equivalent in this context, I would call this field just `validity`, or `validitySeconds` to make the unit explicit.